stdpopsim/cli.py

```diff
@@ -420,6 +423,42 @@ def run_simulation(args):
     species_parser.set_defaults(runner=run_simulation)
 
 
+def dry_run(engine, model, contig, samples, seed=None):
+    # Setup writer
+    printerr = functools.partial(print, file=sys.stderr)
+    indent = " " * 4
+    # Header
+    dry_run_text = "Simulation information:\n"
+    # Get information about the engine
+    dry_run_text += f"{indent}Engine: {engine.id} ({engine.get_version()})\n"
+    # Get information about model
+    dry_run_text += f"{indent}Model id: {model.id}\n"
+    dry_run_text += f"{indent}Model name: {model.name}\n"
+    # Get information about relevant contig
+    gmap = "None" if contig.genetic_map is None else contig.genetic_map.name
+    mean_recomb_rate = contig.recombination_map.mean_recombination_rate
+    mut_rate = contig.mutation_rate
+    contig_len = contig.recombination_map.get_length()
+    dry_run_text += f"{indent * 2}Contig length: {contig_len}\n"
```

Comment:
Why is the contig information indented here?  It makes it look like this is sub-information of the demographic model, but it's really top-level information, I think.